deploy/local/live-csc/docker-compose.yml

```diff
@@ -16,9 +16,9 @@ x-manager-env: &manager-environment
   SERVER_URL: ${SERVER_URL}
   REDIS_HOST: ${REDIS_HOST}
   REDIS_PASS: ${REDIS_PASS}
-  AUTH_LDAP_1_SERVER_URI: ${AUTH_LDAP_1_SERVER_URI}
-  AUTH_LDAP_2_SERVER_URI: ${AUTH_LDAP_2_SERVER_URI}
-  AUTH_LDAP_3_SERVER_URI: ${AUTH_LDAP_3_SERVER_URI}
+  # AUTH_LDAP_1_SERVER_URI: ${AUTH_LDAP_1_SERVER_URI}
```

Comment:
Same here. In general we prefer not to have commented out lines in our code and there suddenly are many in this file. Are these absolutely necessary? If not, please remove them.